README.md

```diff
@@ -59,8 +59,29 @@ final testRun = await TestRun.get(1);
 
 final testSection = await TestSection.get(1);
 ```
+
+Completed or ongoing test run results can be retrieved:
+
+```dart
+final caseResults = await TestResult.getCaseResults(
+  runId: 1833,
+  // Case ID is from TestCases, not TestRun
+  caseId: 184234,
+);
+
+final runResults = await TestResult.getRunResults(
+  runId: 1818,
+  statusId: [5, 1],
+);
+
+final testResults = await TestResult.getTestResults(
+  // Test ID from particular TestRun
+  testId: 1868150,
```

Comment:
This should have a `runId` too right?
```suggestion
  1868150,
```

Reply:
No, that's testId. Because inside run you can get results for a particular test.